services/credential-server-ui/src/pages/ConnectionPage.tsx

```diff
@@ -52,31 +45,34 @@ const ConnectionPage: React.FC = () => {
           <Divider />
           <GetScannerButton />
           <Divider />
-          <GetInputButton />
+          <GetInputButton handleGetContacts={()=> handleGetContacts()}/>
```

Comment:
I want the contacts to refresh once the OOBI has finished resolving. When the OOBI finishes, there's a text saying it completed so after this is displayed. Otherwise it won't have the new contact when we refresh...

Reply:
Isn't it working like this at the moment?

Reply:
yep sorry misready